sdk/include/reactos/mc/neteventmsg.mc

```diff
@@ -1161,6 +1362,9 @@ Language=French
 Language=Polish
 %2: nie można zarejestrować w otoce NDIS.
 .
+Language=Romanian
+%2-ul însuși nu a putut fi înregistrat cu emulatorul NDIS.
```

Comment:
A mere switch to reflexive (and removal of "însuși") in this case should both simplify and convey better the meaning of the English message: "%2 nu s-a putut înregistra..."